tensorflow/python/ops/math_ops.py

```diff
@@ -3288,32 +3304,50 @@ def cumsum(x, axis=0, exclusive=False, reverse=False, name=None):
 
   By default, this op performs an inclusive cumsum, which means that the first
   element of the input is identical to the first element of the output:
+  For example:
 
-  ```python
-  tf.cumsum([a, b, c])  # [a, a + b, a + b + c]
-  ```
-
+  # tf.cumsum([a, b, c])   # [a, a + b, a + b + c]
```

Comment:
Is it possible to make this a doctest too?

Reply:
these were more like syntactic explanations rather than examples, so I don't think they deserve to be doctests.